ReSharper.FSharp/src/FSharp/FSharp.Psi.Features/src/Fsi/FsiHost.fs

```diff
@@ -58,14 +58,7 @@ type FsiHost(lifetime: Lifetime, solution: ISolution, fsiDetector: FsiDetector,
                yield boolArg "shadowcopyreferences" fsiOptions.ShadowCopyReferences.Value
 
                if fsiOptions.SpecifyLanguageVersion.Value then
-                   yield FSharpLanguageVersion.toCompilerArg fsiOptions.LanguageVersion.Value
-
-               if PlatformUtil.IsRunningUnderWindows then
-                   yield stringArg "fsi-server-output-codepage" Encoding.UTF8.CodePage
-                   yield stringArg "fsi-server-input-codepage"  Encoding.UTF8.CodePage
```

Comment:
Yes, you are right! I initially deleted it because it is part of the private Fsi-server settings for VS, which we are not currently using. However, without these settings, the encoding does break. Thank you for your attention!